soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.h

```diff
@@ -10,6 +10,7 @@ extern s16 D_8082AB2C[];
 extern u8 gSlotAgeReqs[];
 extern u8 gEquipAgeReqs[][4];
 extern u8 gAreaGsFlags[];
+extern bool gMaskSelect;
```

Comment:
now i understand the `g`

since this needs to exist beyond the scope of kaleido_item, it probably makes sense to name it `gSelectingMask` or something similar

Reply:
Done, per above. 👍